### PR TITLE
add types definition

### PR DESCRIPTION
**Pull request by taoqf**

### CODE REVIEW

File: lib/dingtalk.d.ts [hunk 1]
```diff
@@ -0,0 +1,268 @@
+declare namespace DingTalk {
+	interface Reponse {
```
Comment (atian25) [inline lib/dingtalk.d.ts]: turn `tab` to 2 spaces ?

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+	"compilerOptions": {
```
Comment (atian25) [inline tsconfig.json]: 2 spaces

**Comment by taoqf** [comment]
Actually, I do not expect it works and would be merged.^v^
Because I find out that the `yeld` and the urllib is so ....

**Comment by atian25** [comment]
so?

**Pull request closed without merging**